### PR TITLE
Enable enforcing turn transport protocol in p2p

### PR DESCRIPTION
**Pull request by pnts-se-whereby**
### Description

**Summary:**

**Related Issue:**

### Checklist

-   [x] My code follows the project's coding standards.
-   [x] Prefixed the PR title and commit messages with the service or package name
-   [ ] I have written unit tests (if applicable).
-   [ ] I have updated the documentation (if applicable).
-   [x] By submitting this pull request, I confirm that my contribution is made
        under the terms of the MIT license.

### CODE REVIEW

File: packages/media/tests/webrtc/P2pRtcManager.spec.ts [hunk 1]
```diff
@@ -591,25 +591,25 @@ describe("P2pRtcManager", () => {
 
     describe("accept", () => {
```
Comment (geirbakke) [inline packages/media/tests/webrtc/P2pRtcManager.spec.ts:592]: acceptNewStream?

**Review by geirbakke — APPROVED** [review]
LGTM 🎉

File: packages/media/tests/webrtc/P2pRtcManager.spec.ts (hunk 1 above)
Comment (pnts-se-whereby) [reply packages/media/tests/webrtc/P2pRtcManager.spec.ts]: Fixed in [media: rename test contextual function](https://github.com/whereby/sdk/pull/378/commits/367ba5b8b0d4fb6fa62129082a0fa8445491e03b)

File: packages/media/src/webrtc/P2pRtcManager.ts [hunk 2]
```diff
@@ -224,10 +225,6 @@ export default class P2pRtcManager implements RtcManager {
         return this._replaceTrackToPeerConnections(oldTrack, newTrack);
     }
 
-    accept({ clientId, shouldAddLocalVideo }: { clientId: string; shouldAddLocalVideo?: boolean }) {
```
Comment (havardholvik) [inline packages/media/src/webrtc/P2pRtcManager.ts:227]: Was this never called then?

Comment (pnts-se-whereby) [reply packages/media/src/webrtc/P2pRtcManager.ts:227]: I did some archeology to try to find out. Maybe way back when SFU and P2P shared the same BaseRtcManager in a meaningful way? If we're using it, there should be a test failing somewhere using it, other than the tests that just verify the implementation.

Comment (havardholvik) [reply packages/media/src/webrtc/P2pRtcManager.ts:227]: Gotcha 👍

**Pull request merged**